packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -80,11 +78,7 @@ export class Analyzer {
       if (ts.isClassDeclaration(statement)) {
         if (isLitElement(statement, this.checker)) {
```

Comment:
Rather than repeating logic between a predicate and the getter, it may be simpler for `getLitElementDeclaration` to return undefined if it's not a LitElement

Reply:
I kind of like how this reads, and it doesn't actually do any duplicate work right now. I could be convinced though. Mind if we defer this? I'll need to change this a little bit to support ReactiveElement, then add a predicate to check for mixins, so this isn't nearly a final shape.

Reply:
Yeah, it's a marginal suggestion, fine to defer